opencti-platform/opencti-front/src/private/components/data/DataTableToolBar.jsx

```diff
@@ -932,9 +937,15 @@ class DataTableToolBar extends Component {
       });
   }
 
-  searchStatuses(i, event, newValue) {
+  searchStatuses(i, selectedTypes, event, newValue) {
     if (!event) return;
     const { actionsInputs } = this.state;
+    let selectedType;
+    if (selectedTypes.length === 1) {
+      [selectedType] = selectedTypes;
+    } else {
+      throw Error('It is not possible to bulk edit statuses if more than one entity type is selected.');
```

Comment:
that would show as an error in the UI ?

Reply:
theoretically, it's not possible to have this error since we check in the code that only one entity type is selected. It's to prevent bad changing of code in the future if someone don't exactly know what he is doing ;)